server/auth.go

```diff
@@ -107,6 +107,9 @@ type AIAuthRequest struct {
 	// Query parameters that came with the stream, if any
 	QueryParams string `json:"query_params,omitempty"`
 
+	// Region in which the gateway is running
+	Region string `json:"region,omitempty"`
```

Comment:
Wouldn't supplying a URL to the gateway be more direct? With only the region, you would still have to infer the ultimate gateway endpoint somehow, and this may not generalize to non-livepeer providers

Reply:
Yes! We were waiting for `gatewayHost` to be available here https://github.com/livepeer/livepeer-infra/pull/2568 and region was just a placeholder while blocked, I am updating now this PR